crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -328,6 +328,48 @@ where
                     guard.on_chain_sync_interval
                 });
 
+                let channels = match node_storage.all_channels_without_costs() {
```

Comment:
🔧 I would prefer a dedicated task even if this data is related to the on-chain sync. The on-chain sync is critical for many reasons and we've had so many problems in the past with this task. Keeping it simple can only help us.